internal/policy_evaluation.go

```
@@ -0,0 +1,128 @@
+func PrefixEvidenceTitles(evidences []*proto.Evidence, prefix string) {
+	prefix = strings.TrimSpace(prefix)
+	if prefix == "" {
+		return
+	}
+
+	for _, evidence := range evidences {
+		if evidence == nil {
+			continue
+		}
+
+		title := strings.TrimSpace(evidence.Title)
```

golangci-lint (2.12.2)

[error] avoid direct access to proto field evidence.Title, use evidence.GetTitle() instead

(protogetter)